services/openvpn/service/manager.go

```diff
@@ -130,28 +134,40 @@ func (m *Manager) Serve(providerID identity.Identity) (err error) {
 		}
 	}()
 
-	if err = m.vpnServer.Start(); err != nil {
-		return
+	if err := m.startServer(m.vpnServer, stateChannel); err != nil {
+		return errors.Wrap(err, "failed to start Openvpn server")
+	}
+
+	m.dnsServer = dns.NewServer("10.8.0.1:53", dns.ResolveViaConfigured())
```

Comment:
"10.8.0.1/24" is just a default value, this subnet could be changed, we should respect it for DNS too.

Reply:
Improved